src/main.cpp

```diff
@@ -954,6 +942,13 @@ int main ( int argc, char** argv )
 #    ifndef HEADLESS
             if ( bUseGUI )
             {
+                // load translation
+                if ( bUseTranslation )
```

Comment:
This will in theory remove the ability to translate CLI output, even if we might not use it yet/it doesn't work currently.

Reply:
It's for consistency, really.  Also, I don't think it was intentional to have it outside the `bUseGUI` block.

Reply:
I think that we once discussed this, but I'd probably be ok with it as is.